_studio/mfx_lib/encode_hw/h264/include/mfx_h264_encode_hw_utils.h

```diff
@@ -2044,6 +2048,7 @@ namespace MfxHwH264Encode
         std::list<DdiTask>  m_histRun;
         std::list<DdiTask>  m_histWait;
         std::list<DdiTask>  m_encoding;
+        std::list<mfxU64>   m_decodeTimeStamp;
```

Comment:
Please rename (e.g. to m_decodeTimeStamps). Otherwise same name in DDITask class will cause problems during reading code/debugging.